### DISCUSSION TITLE
Workflow for textured LOD's

### DISCUSSION THREAD

**Discussion by trojanGoat** [alicevision/Meshroom#1330] [Q&A]
Hey All, 

Can anyone share a work flow they are using to generate different levels of detail and use the same textures?  I know I can make separate nodes to remesh/resamble, but I haven't figured out how to keep the same uv's.

Thanks.

**Comment by natowi** [comment]
I am not aware of a specific lod workflow. At the moment external programs are being used for that. (Blender, CryEngine, UnrealEngine, https://instalod.com/fsl/,....)
